.github/workflows/docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish docker images
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
This makes sure we test if the image can be built on pull request. But we don't publish it.